src/Themes/VuetifyTheme/Views/Content-VList.liquid

```diff
@@ -0,0 +1,16 @@
+<v-list>
+  {% for item in Model.Content.ListPart.ContentItems %}
+      {{ item | shape_build_display | shape_render }}
+      <v-divider inset></v-divider>
```

Comment:
The divider isn’t necessary after every item. Maybe it could be another possible content type that can be added in the list. That way list items could be grouped.

For example, the list could look like:

Apple      (as v-list-item)
Banana   (as v-list-item)
Orange   (as v-list-item)
————   (as v-divider)
Carrot     (as v-list-item)
Celeri      (as v-list-item)
Lettuce   (as v-list-item)